lib/doorkeeper/grape/helpers.rb

```diff
@@ -0,0 +1,44 @@
+require 'doorkeeper/grape/authorization_decorator'
+
+module Doorkeeper
+  module Grape
+    module Helpers
+      extend ::Grape::API::Helpers
+      include Doorkeeper::Rails::Helpers
+
+      # endpoint specific scopes > parameter scopes > default scopes
+      def doorkeeper_authorize!(*scopes)
+        endpoint_scopes = env['api.endpoint'].options[:route_options][:scopes]
+        scope_string = Doorkeeper::OAuth::Scopes.from_array(scopes).all.inspect if scopes && !scopes.empty?
+        scope_string = Doorkeeper::OAuth::Scopes.from_array(endpoint_scopes).all.inspect if endpoint_scopes
+        scopes = Array.wrap(scope_string) if scope_string
```

Comment:
Let's use the block level `if` style, better.

Also, why do you use `.inspect`?